drivers/usb/device/usb_dc_stm32.c

```diff
@@ -453,6 +453,13 @@ static int usb_dc_stm32_init(void)
 		return -EIO;
 	}
 
+	LOG_DBG("HAL_PCD_Stop");
```

Comment:
Please add a quick comment so we can see quickly what was the reason stop is required before start.

Reply:
Done, thanks for your review :)